components/Sections/ImageTextColumns/ImageTextColumns.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import prisma, { Prisma } from "@/lib/prisma";
+import { ImageTextColumnsImageBuilder } from "./ImageTextColumnsImage";
+
+export type ImageTextColumnsProps = Prisma.ImageTextColumnsGetPayload<{}> & {
+  leftImageComponent: React.ReactNode;
+  rightImageComponent: React.ReactNode;
```

Comment:
Rename `leftImageComponent` to `leftImageSlot` and the same for the right image.